src/components/period/PeriodShortcut.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <pkt-button
+    :class="['period-shortcut', { 'pkt-btn--active': active }]"
+    skin="secondary"
+    @onClick="$emit('click', $event)"
+  >
+    <span class="pkt-txt-14">{{ label }}</span>
+    <span v-if="formattedPeriod" class="pkt-txt-12-light">
```

Comment:
Dette er smått ass, på grensen til uleselig for meg, men kanskje bare jeg som har gammelmannsøyne. 😜